agent/app/agent_capability.go

```diff
@@ -553,6 +558,10 @@ func (agent *ecsAgent) appendEBSTaskAttachCapabilities(capabilities []types.Attr
 	return capabilities
 }
 
+func (agent *ecsAgent) appendEBSTANonRootUserCapabilities(capabilities []types.Attribute) []types.Attribute {
```

Comment:
We must only advertise this capability on supported platforms. For example, if Windows is not supported, Windows container instances should not advertise this capability.